wordstats/language_data/t2l/code/getDictonary.sh

```diff
@@ -0,0 +1,65 @@
+# Explanations need to be added and the code as a whole still needs to be refactored
```

Comment:
Victor did that part even if I uploaded it. He told me that he did not have permissions to push anything. I think he sent you a message but I am not sure.